CMakeLists.txt

```diff
@@ -20,13 +20,27 @@ dmlccore_option(USE_OPENMP "Build with OpenMP" ON)
 dmlccore_option(USE_CXX14_IF_AVAILABLE "Build with C++14 if the compiler supports it" OFF)
 dmlccore_option(GOOGLE_TEST "Build google tests" OFF)
 dmlccore_option(INSTALL_DOCUMENTATION "Install documentation" OFF)
+dmlccore_option(USE_SANITIZER "Use santizer flags" OFF)
+dmlccore_option(SANITIZER_PATH "Path to sanitizes." "")
```

Comment:
Some future discussion here. TVM has already adopted a style that unifies USE_XXX and XXX_PATH: if USE_XXX is “on”, it searches automatically; otherwise if it is not “off”, it means a manually set path.

Another thing, though I am not confident (because I am not an expert in CMake), I would love to mention is, what if a upstream project includes DMLC core, but they have some options overlap? For example, both of them have USE_LLVM so they conflict. Our practice is to add prefix to USE_XXX, like PROJECT_NAME_USE_XXX to avoid using the same name. Is there any better way to deal with this?

Reply:
I will add a prefix so that `USE_SANITIZER` doesn't have a name clash